modules/installation-creating-gcp-control-plane.adoc

```diff
@@ -26,11 +26,7 @@ might have to contact Red Hat support with your installation logs.
 
 .Prerequisites
 
-* Configure a GCP account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VPC and associated subnets in GCP.
-* Create and configure networking and load balancers in GCP.
-* Create control plane and compute roles.
+* Ensure you defined the variables in the _Exporting common variables_ section.
```

Comment:
FYI some variables referenced in this section are not defined in "Exporting common variables" section. So?

For example, 
- `CONTROL_SUBNET` , `ZONE_0`, `ZONE_1` and `ZONE_2` are defined in "Creating load balancers in GCP" section.
- `CLUSTER_IMAGE` is defined in "Creating the bootstrap machine in GCP" section.
- `MASTER_SERVICE_ACCOUNT` is defined in "Creating IAM roles in GCP" section.